src/main/cpp/src/get_json_object.cu

```diff
@@ -400,10 +402,10 @@ __device__ thrust::pair<bool, cudf::size_type> evaluate_path(
     // no need to check stack is full
     // because Spark-Rapids already checked maximum length of `path_instruction`
     auto& ctx          = stack[stack_size++];
-    ctx.g              = _g;
+    ctx.g              = std::move(_g);
     ctx.path           = _path;
     ctx.dirty          = 0;
-    ctx.case_path      = _case_path;
+    ctx.case_path      = std::move(_case_path);
```

Comment:
`case_path` is a enum, need a move?

Reply:
Oh yes, not needed.

Reply:
That's my mistake. I intentionally wanted to move `_path` instead.